experiment_scheduler/task_manager/task_manager_server.py

```diff
@@ -223,18 +224,21 @@ def _get_process_return_code(self, task_id: str):
     def get_task_log(self, request, context):
         """
         Save an output of the requested task and return output file path.
-        If status of the requeest task is Done, delete it from task manager.
+        The response is sent by streaming.
         """
-        target_process = self._get_task(request.task_id)
-        if target_process is None:
-            return TaskStatus(logfile_path="")
-
-        log_file_path = osp.join(self.log_dir, f"{request.task_id}_log.txt")
-
-        if self._get_process_return_code(request.task_id) is not None:
-            del self.tasks[request.task_id]
-
-        return TaskLog(logfile_path=log_file_path)
+        log_file_path = osp.join(request.log_file_path, f"{request.task_id}_log.txt")
+        try:
+            f = open(log_file_path, mode="rb")
+            while True:
+                chunk = f.read(CHUNK_SIZE)
+                if chunk:
+                    yield TaskLogFile(log_file=chunk)
+                else:
+                    return
+        except OSError:
+            error_message = f"Getting the log for {request.task_id} fail"
+            logger.error(error_message)
+            yield TaskLogFile(log_file=None, error_message=bytes(error_message, 'utf-8'))
```

Comment:
Add the try-exception.
Because there is no way like do-while in Python, we use 'While True' statement.